Doc/using/windows.rst

```diff
@@ -23,8 +23,9 @@ available for application-local distributions.
 
 As specified in :pep:`11`, a Python release only supports a Windows platform
 while Microsoft considers the platform under extended support. This means that
-Python |version| supports Windows Vista and newer. If you require Windows XP
-support then please install Python 3.4.
+Python |version| supports Windows 8.1 and newer. If you require Windows XP
+support then please install Python 3.4. If you require Windows Vista or
+Windows 7 then please install Python 3.8.
```

Comment:
Technically, Python 3.6 was the last version to officially support Windows Vista, for which Microsoft's extended support ended on 2017-04-11. I don't think 3.7 or 3.8 is using any features that can't at least be supported in an updated installation of Windows Vista. For example, 3.8 uses new loader functionality that's backported to Windows Vista and Windows 7 by [KB2533623](https://support.microsoft.com/kb/2533623). But I'd rather that the docs adhere strictly to PEP 11. In practice it's not much of an issue since the number of people actually using Vista is vanishingly small.

Reply:
Thank you for a review. I fixed supported version for Windows Vista. If the correction intention is wrong, please let me know.